util/dvsim/SGE.py

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env python
```

Comment:
Hi , (I'm Amir Duvdevani work with Sharon on SunGrid dvsim integration)

This Is shebang to use for python scripts, That way you search your environment for the python interpreter to Exactly as you using it in your script 
		dvsim.py:#!/usr/bin/env python3

If you want we can change the shebang to point to python3

Reply:
a

Reply:
Hi , I'm Amir Duvdevani work with Sharon on SunGrid dvsim integration.

This Is shebang to use for python scripts, That way you search your environment for the python interpreter to Exactly as you use it in your script 
		dvsim.py:#!/usr/bin/env python3
If you want we can change the shebang to point to python

<img width="710" alt="Screen Shot 2021-10-27 at 8 03 16 PM" src="https://user-images.githubusercontent.com/94854906/142892857-fe2201f5-9003-44d2-9572-debe0a247fe7.png">
3

Reply:
My guess is that Sri is on a machine without Python 2 installed, where there is no "python" executable at all. Since this is Python 3 code, the correct shebang line is `#!/usr/bin/env python3`.

Reply:
No, I mean this is not a `main` app, so there is no need to add the shebang line.

Reply:
fixed .